numpy/tests/test_ctypeslib.py

```diff
@@ -260,3 +260,9 @@ def test_reference_cycles(self):
             b = np.ctypeslib.as_array(newpnt, (N,))
             # now delete both, which should cleanup both objects
             del newpnt, b
+
+    def test_segmentation_fault(self):
+        a = np.ctypeslib.as_ctypes(np.zeros((224,224,3)))
```

Comment:
Seems this array has to be around the size of `np.zeros(2**20, np.uint8)` for this error to happen